README.md

```diff
@@ -1,3 +1,24 @@
+***
+# NOTICE
+## This package is now _scoped_. We no longer publish new versions here.
+
+Newer versions can be installed by referencing the scoped name `@cloudant/cloudant`, in __npm install__:
```

Comment:
Can we make this a link to the scoped repo?

Reply:
yup